tests/unit/Main.spec.js

```diff
@@ -0,0 +1,95 @@
+jest.mock("../../src/fetchedValues/agency.json", () => ({}), { virtual: true });
+jest.mock("../../src/fetchedValues/states.json", () => ({}), { virtual: true });
+jest.mock("../../src/fetchedValues/siteTypes.json", () => ({}), {
+  virtual: true
+});
+jest.mock("../../src/fetchedValues/counties.json", () => ({}), {
+  virtual: true
+});
+
+jest.mock(
+  "../../src/fetchedValues/locAquifer.json",
+  () => {
+    return [];
+  },
+  { virtual: true }
+);
+
+jest.mock("../../src/fetchedValues/fips.json", () => ({}), { virtual: true });
+
+jest.mock("../../src/fetchedValues/aquiferAreas.json", () => ({}), {
+  virtual: true
+});
+
+jest.mock(
+  "../../src/fetchedValues/timezones.json",
+  () => {
+    return [];
+  },
+  { virtual: true }
+);
+
+jest.mock(
+  "../../src/fetchedValues/paramTypes.json",
+  () => {
+    return [];
+  },
+  { virtual: true }
+);
+
+import { shallowMount, createLocalVue } from "@vue/test-utils";
+import Main from "../../src/components/Main.vue";
+import Vuex from "vuex";
```

Comment:
You may also want to try mocking the commit methods from Vuex.

Reply:
In my new test branch, I've replaced some commit functions with stubs when the commits are called, as as to prevent an unknown commit warning. This does not occur to the best of my knowledge in main.vue.